test-js/integration.js

```diff
@@ -236,6 +237,17 @@ describe("integration tests", function() {
     assert.equal(out.trim(), test);
   }));
 
+  it("pulp test --test-path test2 -- --something-node-wouldnt-like", run(function*(sh, pulp, assert, temp) {
+    const newTest = path.join(temp, "test2");
+    yield pulp("init");
+    yield fs.mkdir(newTest);
+    yield fs.copy(path.resolve(__dirname, "resources/Main.purs"), path.join(newTest, "Main.purs"));
+    yield fs.copy(path.resolve(__dirname, "resources/Main.js"), path.join(newTest, "Main.js"));
+    yield sh("bower i purescript-arrays --save");
```

Comment:
Can you think of a good way of avoiding the need to invoke `bower` here? I'd like to make it as easy as possible to switch to `psc-package` once we are ready to.